### ISSUE TITLE
Seeking Accessible Alternative to Select 2 for Drupal Version 10.4

### ISSUE THREAD

**Issue by Rahulch-php** [select2/select2#6356]
We are hoping for recommendations for an accessible alternative to Select 2 for Drupal Version 10.4. My organization is big on accessibility, and our accessibility tool, Siteimprove, is flagging our Month and Profession multiselect options on our site - [Enforcement Actions | Office of the Professions](https://www.op.nysed.gov/enforcement/enforcement-actions) - as inaccessible ([Accessibility page report for Enforcement Actions | Office of the Professions](https://my2.siteimprove.com/Inspector/327230/A11Y/Page?pageId=99719889666&impmd=F3E542C1549A6FBC0E1E482C6913B259&conf=a+aa+aaa+aria+si&issue=cantTell+failed&wcag=twopointtwo&decision=dismissed+falsePositive+none#/sia-r8/failed)). I posted the full issue on the Select 2 Drupal Issue page - https://www.drupal.org/project/select2/issues/3494619 - but we aren't receiving responses from the community. I also reached out to the moderators, but they haven't responded to me either. Any guidance would be appreciated.
